ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -1272,25 +1280,30 @@ def test_torch_svd(
         fn_tree=fn_tree,
         on_device=on_device,
         test_values=False,
-        atol=1e-03,
-        rtol=1e-05,
         A=x,
         full_matrices=full_matrices,
     )
     ret = [ivy.to_numpy(x) for x in ret]
```

Comment:
I think the problem is caused by this line, maybe change it to `ret = [np.asarray(x) for x in ret]` and see if that works?